shell/app/agent.js

```diff
@@ -134,8 +134,8 @@ const reqKeyList = [];
 //   }
 // }
 
-agent.use(setHeader);
 agent.use(handleSpotPrefix);
+agent.use(setHeader);
```

Comment:
Should not count on plugin use order which is fragile. It's better to move all modify url logic to a `updateUrl` agent plugin.